flow/metrics.yaml

```diff
@@ -19,6 +19,13 @@ components:
           type: boolean
           default: false
           x-field-uid: 2
+        rx_tx_ratio:
+          description: |-
+            Tx Rx multiplication factor, required for loss measurement. Should be a positive, non-zero value.
```

Comment:
Also, a more detailed explanation here would be useful, especially detailing the scenario when Rx packets = Tx packets * number of Rx ports for multicast or broadcast traffic.  
Finally , we could give some  explanation of scenarios when custom and non-integral values are needed 
e.g. there are multiple dst IPs configured within one flow , but DUT is configured to replicate only to a subset of Rx ports.
Or Tx side generates two packets from each source in 1:1 protection  but only one of the two packets are received by Rx ports ( so replication factor becomes 0.5 and not an integral number )

Reply:
Descriptions also updated for each enum value.